schema/transportation/segment.yaml

```diff
@@ -258,17 +257,18 @@ properties:
                        - gatedKey
                      default:
                        enum: [legal]
-                   direction:
-                     description: direction of travel on the source that pertains to the relation
-                     "$ref": "#/$defs/propertyDefinitions/direction"
-                   via:
+                   sequence:
                      description: Sequence of ordered segements that needs to be traversed in order for the relation to be effective
```

Comment:
nits:
1. Segment has two e's, not three. (Not seg*e*ment...)
2. It's an *ordered sequence* of segments, not a sequence of *ordered segments*.

----
I would be tempted to write this as:

> Ordered sequence of segments that may not be traversed, where the last segment in the list is the ultimate destination of the transition.

Reply:
Interesting thing about segement - found many instances and not all of them are mine. I guess it has something to do with muscle memory - g followed by e :)

Reply:
Fixed